3p/facebook.js

```diff
@@ -87,6 +87,8 @@ function getPageContainer(global, data) {
   container.setAttribute('data-small-header', data.smallHeader);
   container.setAttribute(
       'data-adapt-container-width', data.adaptContainerWidth);
+  container.setAttribute('data-width', data.width);
```

Comment:
I believe (not tested) that if we always set `data-adapt-container-width` to `true`. things should just work and we don't need to propagate width and height. Since AMP controls the layout of the container, telling facebook to "fill the width" should behave correctly in all our layouts. I might be wrong but worth a shot.

Reply:
Hmm in that case we should only set it to true if the user hasn't set it to a specific value themselves. Making the change.

Reply:
I don't think we should allow authors to set it at all. Since AMP controls the size of the container, it should always be set to true. let's remove it from the documentation if it is there.

Reply:
That makes sense. On it! Thanks for the insight. That lines up with why the way the other elements behave.

Reply:
Just tried that and for the following code: 
`
<amp-facebook-page width="200" height="200"
    layout="responsive"
    data-show-facepile="true"
    data-href="https://www.facebook.com/itsdougthepug">
</amp-facebook-page>
`
the results are:

*With `data-adapt-container-width` set to 'true':*
<img width="380" alt="screen shot 2018-02-23 at 3 36 35 pm" src="https://user-images.githubusercontent.com/7919520/36578185-ab43687e-18af-11e8-9718-40f4440a6aeb.png">
In this case while the `data-adapt-container-width` is propagating through the width and height are the container's width and height

*With `data-width` and `data-height` propagated through:*
<img width="283" alt="screen shot 2018-02-23 at 3 40 59 pm" src="https://user-images.githubusercontent.com/7919520/36578237-ffd1cd18-18af-11e8-8097-bc645cd55676.png">
The width and height are being deliberately set here to the value specified by the user.

Reply:
First one is correct for layout=responsive since  width and height simply dictate the aspect ratio with responsive layout. The component also resizes which reduces the height to what's needed. Try the first case with layout=fixed and then width should be exactly 200

Reply:
Yup tested that - this is ready to review now.